src/api_utils/gladia_api_utils/system.py

```diff
@@ -72,3 +73,20 @@ def remove(*paths):
                 shutil.rmtree(path)
             else:
                 os.remove(path)
+
+
+def get_first_available_gpu_id() -> int:
+    gpu_id = None
+    if get_available_gpu_ids():
+        gpu_id = get_available_gpu_ids()[0]
```

Comment:
You are two two times the computation:
```suggestion
available_gpu_id = get_available_gpu_ids()

return None if len(available_gpu_ids) == 0 else available_gpu_ids[0]
```